### DISCUSSION TITLE
Unidirectional Lightning Infill

### DISCUSSION THREAD

**Discussion by humanmilkfactory** [OrcaSlicer/OrcaSlicer#5402] [Ideas]
Is there any way to make lightning infill affect all walls and not just the top one? I understand the infill is designed for speed, but I'm more interested for the strength to weight ratio benefits and its potential as a gradient infill. 

EDIT: Ugh I meant omnidirectional.

**Comment by silvernull** [comment]
I think lightning generating for all overhangs could be interesting also.
